frontend/src/app/services/covid.service.ts

```diff
@@ -45,6 +44,23 @@ export class CovidService {
   }
 
 
+  public async getComparisonData(): Promise<any> {
```

Comment:
Currently the areaId is handed over as dummy data with the "params" value in the url. So all further usages of areaId are implemented. I thought it would be easier and more understandable if I leave it out until now and that we could implement it with the issue "flexible charts" as it would be a simple copy&paste from Manuels "Compare Regions" chart....

But if you want I could add it though.

Reply:
I think it would be nice to add the parameter here and use the dummy data in the initializeComparisonCasesChart method. The multiselect has not to be added.
When the parameter is already there, the multiselect is more easily implemented i think.